open-vm-tools-centos/service.template

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service for virtual machines hosted on VMware
+Documentation=http://github.com/vmware/open-vm-tools
+ConditionVirtualization=vmware
```

Comment:
So this container would only work in a vmware virtualized environment? In that case it's probably best to note somewhere in README